sc-memory/sc-memory/sc_memory.cpp

```diff
@@ -336,9 +336,15 @@ ScStreamPtr ScMemoryContext::GetLinkContent(ScAddr const & addr)
 
   sc_stream * s = nullptr;
   if (sc_memory_get_link_content(m_context, *addr, &s) != SC_RESULT_OK)
+  {
+    SC_THROW_EXCEPTION(utils::ExceptionCritical,
```

Comment:
This function should not throws an exception. It should return null ptr. My comment was about throwing exception in `ScLink::Type Get() const` that already exist